src/main/java/com/example/carservice/service/CarServiceImpl.java

```diff
@@ -11,12 +11,26 @@ public class CarServiceImpl implements CarService {
 
     private final CarRepository repository;
 
-    public CarServiceImpl(CarRepository repository) {
+
+    public CarServiceImpl(CarRepository repository, CarRepository carRepository) {
         this.repository = repository;
     }
 
+    public Car findCarByModel() {
+        Car carToUpdate = repository.findCarByModel("IS-250");
+        return carToUpdate;
+    }
+
     @Override
-    public List<Car> getCars() {
+    public List<Car> getAll() {
         return repository.findAll();
     }
+
+    @Override
+    public Car update() {
+        Car carToUpdate = findCarByModel();
+        carToUpdate.setColor("black");
+        repository.save(carToUpdate);
```

Comment:
Lootk at save() method. (Ctrl + B)

Save return the same object, but with updated fields.
I mentioned it. Be careful.